### PR TITLE
Add an Icon component

### PR DESCRIPTION
**Pull request by sisou**
The various icons can be shown with a property, which the component uses to dynamically display the respective SVG component.

Please refer to #28 for prior discussion.

### CODE REVIEW

**Comment by sisou** [comment]
We could do that right now, by just moving the imports in Icon.vue onto the main.ts file and exporting all the svg. The loader converts them into components during compile, done.

And we could still have the general Icon component that combines all icons. 

The problem with the direct svg component exports is, that they would not have the `nq-icon` class. There is no way to configure the loader to add a class. The source looks like it would be able to use a class set on the imported svg tag, but I don't know if we should put that class on the nimiq/style icon source.

**Comment by Bettelstab** [comment]
Since we even style the svgs with CSS coming from `@nimiq/style`, not from `vue-components`, I would argue that it makes sense to include that class there.

File: src/components/Icon.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<template>
+    <component class="nq-icon" :is="componentName"/>
+</template>
+
+<script lang="ts">
+import {Component, Prop, Vue} from 'vue-property-decorator';
+
+import AlertTriangleIcon from '@nimiq/style/src/icons/alert-triangle.svg';
```
Comment (danimoh) [inline src/components/Icon.vue]: A comment here that these svgs are imported as components via vue-sv-loader would be nice.

File: src/components/Icon.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import SkullIcon from '@nimiq/style/src/icons/skull.svg';
+import TransferIcon from '@nimiq/style/src/icons/transfer.svg';
+import ViewOffIcon from '@nimiq/style/src/icons/view-off.svg';
+import ViewIcon from '@nimiq/style/src/icons/view.svg';
+
+/**
+ * # Icons
+ *
+ * Comment out the ones not used for the build target.
+ */
```
Comment (danimoh) [inline src/components/Icon.vue]: And a comment here that webpack will remove the unused imports.

**Comment by Bettelstab** [comment]
How do you like this: 
https://github.com/nimiq/vue-components/commit/d36d15715cf26fea65cb90edc4fb048e3d19e7f0

**Comment by sisou** [comment]
I was really hoping to not create individual files for each icon...

**Comment by Bettelstab** [comment]
I see, well, this approach also works with one file of course.

**Pull request merged**